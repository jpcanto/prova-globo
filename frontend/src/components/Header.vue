<template>
  <v-main class="mb-10 elevation-3">
    <v-app-bar color="grey lighten-3" dark flat height="100">
      <template v-if="!isDesktop">
        <v-btn light outlined class="mx-2" large color="pink darken-1" @click.stop="menu = !menu">
          <v-icon color="pink darken-1">mdi-menu</v-icon>
        </v-btn>
      </template>
      <v-container v-if="isDesktop" class="ma-0 align-center" fill-height fluid>
        <v-btn class="btn-gradient"></v-btn>

        <v-divider class="mx-8" color="grey" inset vertical></v-divider>

        <v-btn light class="btn">
          <v-icon color="grey darken-2">mdi-security</v-icon>
        </v-btn>
        <v-btn class="btn btn-gradient mr-5">
          <v-icon color="white">mdi-account</v-icon>
        </v-btn>

        <form @submit.prevent="setSearchBarFilter" class="mt-3">
          <v-text-field
            v-model="search"
            light
            label="Pesquisar..."
            class="mx-4"
            append-icon="mdi-magnify"
            @click:append="setSearchBarFilter"
          >
          </v-text-field>
        </form>

        <v-spacer></v-spacer>

        <v-btn light @click.stop="drawer = !drawer" class="btn mr-4 filter">
          <v-icon color="grey darken-1">mdi-tune</v-icon>
        </v-btn>

        <v-btn
          elevation="1"
          tile
          color="pink darken-1"
          class="btn user-btn text-body-2 pl-0 mr-6"
          @click="handleUser"
        >
          <v-icon class="ma-2 create-user">mdi-account </v-icon>
          INCLUIR USUÁRIO
        </v-btn>

        <v-divider class="mx-5" color="grey" inset vertical></v-divider>

        <v-btn icon>
          <v-icon color="grey darken-1" class="btn">mdi-home</v-icon>
        </v-btn>
        <v-btn icon>
          <v-icon color="grey darken-1" class="btn" @click="handleApiInfo">mdi-cog</v-icon>
        </v-btn>
        <v-btn icon>
          <v-icon color="grey darken-1" class="btn">mdi-power</v-icon>
        </v-btn>
      </v-container>
    </v-app-bar>

    <v-navigation-drawer v-model="drawer" fixed right temporary color="grey lighten-3" width="30vw">
      <v-toolbar color="grey lighten-5" height="90" class="elevation-1">
        <v-icon color="grey darken-1" class="btn mr-2">mdi-tune</v-icon>
        <v-toolbar-title>FILTROS</v-toolbar-title>
        <v-btn small absolute right color="elevation-1" @click.stop="drawer = !drawer">X</v-btn>
      </v-toolbar>
      <div class="grey--text font-italic ma-10">
        Utilize os filtros abaixo para refinar os resultados da tabela, clique no botão APLICAR para
        salvar as alterações.
      </div>
      <v-container class="mt-7">
        <v-select
          v-model="inclusion"
          :items="inclusionItems"
          menu-props="auto"
          label="TODAS AS DATAS DE INCLUSÃO"
          prepend-inner-icon="mdi-calendar"
          color="pink darken-1"
          background-color="grey lighten-3"
          height="60"
          filled
          class="pl-4 pr-4"
        ></v-select>
        <v-select
          v-model="alteration"
          :items="alterationItems"
          menu-props="auto"
          label="TODAS AS DATAS DE ALTERAÇÃO"
          prepend-inner-icon="mdi-calendar"
          color="pink darken-1"
          background-color="grey lighten-3"
          height="60"
          filled
          class="pl-4 pr-4"
        ></v-select>
        <v-select
          v-model="actives"
          :items="activeItems"
          menu-props="auto"
          label="ATIVOS E INATIVOS"
          prepend-inner-icon="mdi-dots-horizontal"
          color="pink darken-1"
          background-color="grey lighten-3"
          height="60"
          filled
          class="pl-4 pr-4"
        ></v-select>
      </v-container>
      <v-btn
        x-large
        outlined
        width="-webkit-fill-available"
        color="pink darken-1"
        class="ma-8"
        @click="setComboFilters"
      >
        APLICAR
      </v-btn>
    </v-navigation-drawer>

    <v-navigation-drawer v-model="menu" fixed right temporary width="30vw">
      <v-toolbar color="grey lighten-5" height="90" class="elevation-1">
        <v-toolbar-title>MENU</v-toolbar-title>
        <v-btn small absolute right color="elevation-1" @click.stop="menu = !menu">X</v-btn>
      </v-toolbar>
      <v-btn
        elevation="1"
        tile
        color="pink darken-1"
        outlined
        class="btn user-btn text-body-2 pl-0 ma-6"
        @click="handleUser"
      >
        <v-icon class="ma-2 create-user">mdi-account </v-icon>
        INCLUIR USUÁRIO
      </v-btn>

      <v-btn
        elevation="1"
        tile
        color="pink darken-1"
        outlined
        class="btn user-btn text-body-2 pl-0 ma-6"
        @click.stop="
          drawer = !drawer;
          menu = !menu;
        "
      >
        <v-icon class="ma-2 create-user">mdi-tune </v-icon>
        FILTROS
      </v-btn>

      <form @submit.prevent="setSearchBarFilter" class="mt-3">
        <v-text-field
          v-model="search"
          light
          outlined
          clearable
          label="Pesquisar..."
          color="pink darken-1"
          class="mx-4"
          append-icon="mdi-magnify"
          @click:append="setSearchBarFilter"
        >
        </v-text-field>
      </form>
    </v-navigation-drawer>
  </v-main>
</template>

<script>
import { mapGetters, mapState } from 'vuex';

export default {
  name: 'Header',
  data() {
    return {
      isDesktop: this.$vuetify.breakpoint.mdAndUp,
      inclusionItems: [],
      alterationItems: [],
      activeItems: [],
      inclusion: [],
      alteration: [],
      actives: [],
      search: '',
      drawer: false,
      menu: false
    };
  },
  computed: {
    ...mapState({
      users: state => state.users
    }),
    ...mapGetters({
      newInclusionItems: 'getInclusionItems',
      newAlterationItems: 'getAlterationItems',
      newActiveItems: 'getActiveItems'
    })
  },
  watch: {
    newInclusionItems: function(value) {
      this.inclusionItems = value;
    },
    newAlterationItems: function(value) {
      this.alterationItems = value;
    },
    newActiveItems: function(value) {
      this.activeItems = value;
    }
  },
  methods: {
    setComboFilters() {
      console.log({
        inclusionDate: this.inclusion,
        alterationDate: this.alteration,
        status: this.actives
      });
      this.$store.commit('setUsersTableFilterCombo', {
        inclusionDate: this.inclusion,
        alterationDate: this.alteration,
        status: this.actives
      });

      if (this.drawer) this.drawer = false;
    },
    setSearchBarFilter() {
      this.$store.commit('setFilterUsersTableParam', this.search);
    },
    handleUser() {
      this.$store.dispatch('handleRow', {
        _id: 'default',
        email: 'default',
        name: 'default',
        inclusionDate: 'default',
        alterationDate: 'default',
        rules: 'default',
        status: 'default',
        show: false
      });
      this.$store.commit('toggleCrudDialog');
      this.$store.commit('setDialogType', 'create');
    },
    handleApiInfo() {
      this.$store.commit('toggleCrudDialog');
      this.$store.commit('setDialogType', 'api-info');
    }
  }
};
</script>

<style lang="scss" scoped>
form {
  width: 25vw;
}

@media only screen and (max-width: 770px) {
  .v-sheet.v-app-bar.v-toolbar {
    height: 12vh !important;
  }
  .v-navigation-drawer {
    width: 50vw !important;
  }
}
@media only screen and (max-width: 420px) {
  .v-text-field {
    width: 80vw !important;
  }
  .v-navigation-drawer {
    width: 80vw !important;
  }
}

.v-text-field {
  max-width: 50vw;
}
.btn {
  border-radius: 5px;
  height: 45px !important;
  width: fit-content;

  & .v-icon.v-icon {
    font-size: 20px;
  }
}
.btn-gradient {
  background-image: linear-gradient(to right top, #dc375d, #ea434f, #f3543e, #f8672a, #f97b07);
}
.v-divider--vertical.v-divider--inset {
  border-color: rgb(228 228 228 / 87%);
  border-width: 1px;
  margin-top: 20px;
  max-height: calc(100% - 40px);
}
</style>
