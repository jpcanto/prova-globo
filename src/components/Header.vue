<template>
  <v-main>
    <v-app-bar color="grey lighten-3" dark flat height="100px">
      <v-container class="ma-0 align-center" fill-height fluid>
        <form @submit.prevent="setFilters">
          <v-text-field
            v-model="search"
            light
            label="Pesquisar..."
            class="mx-4"
            append-icon="mdi-magnify"
            @click:append="setFilters"
          >
          </v-text-field>
        </form>

        <v-spacer></v-spacer>

        <v-btn light @click.stop="drawer = !drawer" class="btn mr-4">
          <v-icon color="grey darken-1">mdi-abacus</v-icon>
        </v-btn>

        <v-btn elevation="1" tile color="pink accent-4" class="btn user-btn text-body-2 pl-0 mr-6">
          <v-icon class="ma-2">mdi-account </v-icon>
          INCLUIR USUÁRIO
        </v-btn>

        <v-btn icon>
          <v-icon color="grey darken-1" class="btn">mdi-home</v-icon>
        </v-btn>
        <v-btn icon>
          <v-icon color="grey darken-1" class="btn">mdi-cog</v-icon>
        </v-btn>
        <v-btn icon>
          <v-icon color="grey darken-1" class="btn">mdi-power</v-icon>
        </v-btn>
      </v-container>
    </v-app-bar>

    <v-navigation-drawer v-model="drawer" fixed right temporary width="30vw">
      <v-toolbar class="elevation-1">
        <v-toolbar-title>FILTROS</v-toolbar-title>
        <v-btn small color="elevation-1" @click.stop="drawer = !drawer">X</v-btn>
      </v-toolbar>
      <v-list>
        <v-list-item-group v-model="group" active-class="pink accent-2 text--accent-2">
          <v-list-item>
            <v-list-item-title>TODAS AS DATAS DE INCLUSÃO</v-list-item-title>
          </v-list-item>

          <v-list-item>
            <v-list-item-title>TODAS AS DATAS DE ALTERAÇÃO</v-list-item-title>
          </v-list-item>

          <v-list-item>
            <v-list-item-title>ATIVOS E INATIVOS</v-list-item-title>
          </v-list-item>

          <v-btn x-large outlined color="pink accent-4">APLICAR</v-btn>
        </v-list-item-group>
      </v-list>
    </v-navigation-drawer>
  </v-main>
</template>

<script>
export default {
  data: () => ({
    search: '',
    drawer: false,
    group: null
  }),
  methods: {
    setFilters() {
      this.$store.dispatch('setFilterUsersTableParam', this.search);
    }
  },
  watch: {
    group() {
      this.drawer = false;
    }
  }
};
</script>

<style lang="scss" scoped>
form {
  width: 25vw;
}

.v-text-field {
  max-width: 50vw;
}
.btn {
  border-radius: 5px;
  height: 45px !important;
  width: fit-content;
}

</style>
