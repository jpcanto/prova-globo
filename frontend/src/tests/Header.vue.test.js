import Vuetify from 'vuetify';
import Vuex from 'vuex';
import { mount, createLocalVue } from '@vue/test-utils';

import Header from '../components/Header.vue';

const localVue = createLocalVue();
localVue.use(Vuex);

describe('Home suit', () => {
  let wrapper;

  beforeEach(() => {
    const store = new Vuex.Store({
      state: {
        users: [
          {
            _id: '601f3794e3c17b3814c8f28b',
            email: 'antonio.pina@tvglobo.com.br',
            name: 'ANPINA',
            inclusionDate: '28/05/2019',
            alterationDate: '30/05/2019',
            rules: '01',
            status: 'ATIVO',
            show: false,
            __v: 0
          }
        ]
      }
    });

    const vuetify = new Vuetify();

    wrapper = mount(Header, {
      localVue,
      vuetify,
      store
    });
  });

  it('matches snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });

  it('toggle nav drawer', () => {
    const button = wrapper.find('.filter');
    const drawer = wrapper.find('.v-navigation-drawer');

    expect(button.exists()).toBe(true);
    expect(drawer.classes()).toContain('v-navigation-drawer--close');

    button.trigger('click');
    expect(drawer.classes()).toContain('v-navigation-drawer--fixed');
  });
});
