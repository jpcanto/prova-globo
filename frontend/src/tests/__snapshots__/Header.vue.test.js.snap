// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home suit matches snapshot 1`] = `
"<main class=\\"v-main mb-10 elevation-3\\" style=\\"padding: 0px 0px 0px 0px;\\">
  <div class=\\"v-main__wrap\\">
    <header class=\\"v-sheet theme--dark v-toolbar v-toolbar--flat v-app-bar grey lighten-3\\" style=\\"height: 100px; margin-top: 0px; transform: translateY(0px); left: 0px; right: 0px;\\">
      <div class=\\"v-toolbar__content\\" style=\\"height: 100px;\\">
        <div class=\\"container ma-0 align-center container--fluid fill-height\\"><button type=\\"button\\" class=\\"btn-gradient v-btn v-btn--is-elevated v-btn--has-bg theme--dark v-size--default\\"><span class=\\"v-btn__content\\"></span></button>
          <hr role=\\"separator\\" aria-orientation=\\"vertical\\" color=\\"grey\\" class=\\"mx-8 v-divider v-divider--inset v-divider--vertical theme--dark\\"> <button type=\\"button\\" class=\\"btn v-btn v-btn--is-elevated v-btn--has-bg theme--light v-size--default\\"><span class=\\"v-btn__content\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-security theme--light grey--text text--darken-2\\"></i></span></button> <button type=\\"button\\" class=\\"btn btn-gradient mr-5 v-btn v-btn--is-elevated v-btn--has-bg theme--dark v-size--default\\"><span class=\\"v-btn__content\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-account theme--dark white--text\\"></i></span></button>
          <form class=\\"mt-3\\">
            <div class=\\"v-input mx-4 theme--light v-text-field\\">
              <div class=\\"v-input__control\\">
                <div class=\\"v-input__slot\\">
                  <div class=\\"v-text-field__slot\\"><label for=\\"input-12\\" class=\\"v-label theme--light\\" style=\\"left: 0px; position: absolute;\\">Pesquisar...</label><input id=\\"input-12\\" type=\\"text\\"></div>
                  <div class=\\"v-input__append-inner\\">
                    <div class=\\"v-input__icon v-input__icon--append\\"><button type=\\"button\\" aria-label=\\"append icon\\" class=\\"v-icon notranslate v-icon--link mdi mdi-magnify theme--light\\"></button></div>
                  </div>
                </div>
                <div class=\\"v-text-field__details\\">
                  <div class=\\"v-messages theme--light\\">
                    <transition-group-stub tag=\\"div\\" name=\\"message-transition\\" class=\\"v-messages__wrapper\\"></transition-group-stub>
                  </div>
                </div>
              </div>
            </div>
          </form>
          <div class=\\"spacer\\"></div> <button type=\\"button\\" class=\\"btn mr-4 filter v-btn v-btn--is-elevated v-btn--has-bg theme--light v-size--default\\"><span class=\\"v-btn__content\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-tune theme--light grey--text text--darken-1\\"></i></span></button> <button type=\\"button\\" class=\\"btn user-btn text-body-2 pl-0 mr-6 v-btn v-btn--is-elevated v-btn--has-bg v-btn--tile theme--dark elevation-1 v-size--default pink darken-1\\"><span class=\\"v-btn__content\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate ma-2 mdi mdi-account theme--dark\\"></i>
        INCLUIR USUÁRIO
      </span></button>
          <hr role=\\"separator\\" aria-orientation=\\"vertical\\" color=\\"grey\\" class=\\"mx-5 v-divider v-divider--inset v-divider--vertical theme--dark\\"> <button type=\\"button\\" class=\\"v-btn v-btn--icon v-btn--round theme--dark v-size--default\\"><span class=\\"v-btn__content\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate btn mdi mdi-home theme--dark grey--text text--darken-1\\"></i></span></button> <button type=\\"button\\" class=\\"v-btn v-btn--icon v-btn--round theme--dark v-size--default\\"><span class=\\"v-btn__content\\"><button type=\\"button\\" class=\\"v-icon notranslate btn v-icon--link mdi mdi-cog theme--dark grey--text text--darken-1\\"></button></span></button> <button type=\\"button\\" class=\\"v-btn v-btn--icon v-btn--round theme--dark v-size--default\\"><span class=\\"v-btn__content\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate btn mdi mdi-power theme--dark grey--text text--darken-1\\"></i></span></button>
        </div>
      </div>
    </header>
    <aside class=\\"v-navigation-drawer v-navigation-drawer--close v-navigation-drawer--fixed v-navigation-drawer--is-mobile v-navigation-drawer--right v-navigation-drawer--temporary theme--light grey lighten-3\\" style=\\"height: 100%; top: 0px; transform: translateX(100%); width: 30vw;\\">
      <div class=\\"v-navigation-drawer__content\\">
        <header class=\\"elevation-1 v-sheet theme--light v-toolbar grey lighten-5\\" style=\\"height: 90px;\\">
          <div class=\\"v-toolbar__content\\" style=\\"height: 90px;\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate btn mr-2 mdi mdi-tune theme--light grey--text text--darken-1\\"></i>
            <div class=\\"v-toolbar__title\\">FILTROS</div> <button type=\\"button\\" class=\\"v-btn v-btn--absolute v-btn--is-elevated v-btn--has-bg v-btn--right theme--light v-size--small elevation-1\\"><span class=\\"v-btn__content\\">X</span></button>
          </div>
        </header>
        <div class=\\"grey--text font-italic ma-10\\">
          Utilize os filtros abaixo para refinar os resultados da tabela, clique no botão APLICAR para
          salvar as alterações.
        </div>
        <div class=\\"container mt-7\\">
          <div class=\\"v-input pl-4 pr-4 theme--light v-text-field v-text-field--filled v-text-field--enclosed v-select\\">
            <div class=\\"v-input__control\\">
              <div role=\\"button\\" aria-haspopup=\\"listbox\\" aria-expanded=\\"false\\" aria-owns=\\"list-31\\" class=\\"v-input__slot grey lighten-3\\" style=\\"height: 60px;\\">
                <div class=\\"v-input__prepend-inner\\">
                  <div class=\\"v-input__icon v-input__icon--prepend-inner\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-calendar theme--light\\"></i></div>
                </div>
                <div class=\\"v-select__slot\\"><label for=\\"input-31\\" class=\\"v-label theme--light\\" style=\\"left: 0px; position: absolute;\\">TODAS AS DATAS DE INCLUSÃO</label>
                  <div class=\\"v-select__selections\\"><input id=\\"input-31\\" readonly=\\"readonly\\" type=\\"text\\" aria-readonly=\\"false\\" autocomplete=\\"off\\"></div>
                  <div class=\\"v-input__append-inner\\">
                    <div class=\\"v-input__icon v-input__icon--append\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-menu-down theme--light\\"></i></div>
                  </div><input type=\\"hidden\\">
                </div>
                <div class=\\"v-menu\\">
                  <!---->
                </div>
              </div>
              <div class=\\"v-text-field__details\\">
                <div class=\\"v-messages theme--light\\">
                  <transition-group-stub tag=\\"div\\" name=\\"message-transition\\" class=\\"v-messages__wrapper\\"></transition-group-stub>
                </div>
              </div>
            </div>
          </div>
          <div class=\\"v-input pl-4 pr-4 theme--light v-text-field v-text-field--filled v-text-field--enclosed v-select\\">
            <div class=\\"v-input__control\\">
              <div role=\\"button\\" aria-haspopup=\\"listbox\\" aria-expanded=\\"false\\" aria-owns=\\"list-37\\" class=\\"v-input__slot grey lighten-3\\" style=\\"height: 60px;\\">
                <div class=\\"v-input__prepend-inner\\">
                  <div class=\\"v-input__icon v-input__icon--prepend-inner\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-calendar theme--light\\"></i></div>
                </div>
                <div class=\\"v-select__slot\\"><label for=\\"input-37\\" class=\\"v-label theme--light\\" style=\\"left: 0px; position: absolute;\\">TODAS AS DATAS DE ALTERAÇÃO</label>
                  <div class=\\"v-select__selections\\"><input id=\\"input-37\\" readonly=\\"readonly\\" type=\\"text\\" aria-readonly=\\"false\\" autocomplete=\\"off\\"></div>
                  <div class=\\"v-input__append-inner\\">
                    <div class=\\"v-input__icon v-input__icon--append\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-menu-down theme--light\\"></i></div>
                  </div><input type=\\"hidden\\">
                </div>
                <div class=\\"v-menu\\">
                  <!---->
                </div>
              </div>
              <div class=\\"v-text-field__details\\">
                <div class=\\"v-messages theme--light\\">
                  <transition-group-stub tag=\\"div\\" name=\\"message-transition\\" class=\\"v-messages__wrapper\\"></transition-group-stub>
                </div>
              </div>
            </div>
          </div>
          <div class=\\"v-input pl-4 pr-4 theme--light v-text-field v-text-field--filled v-text-field--enclosed v-select\\">
            <div class=\\"v-input__control\\">
              <div role=\\"button\\" aria-haspopup=\\"listbox\\" aria-expanded=\\"false\\" aria-owns=\\"list-43\\" class=\\"v-input__slot grey lighten-3\\" style=\\"height: 60px;\\">
                <div class=\\"v-input__prepend-inner\\">
                  <div class=\\"v-input__icon v-input__icon--prepend-inner\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-dots-horizontal theme--light\\"></i></div>
                </div>
                <div class=\\"v-select__slot\\"><label for=\\"input-43\\" class=\\"v-label theme--light\\" style=\\"left: 0px; position: absolute;\\">ATIVOS E INATIVOS</label>
                  <div class=\\"v-select__selections\\"><input id=\\"input-43\\" readonly=\\"readonly\\" type=\\"text\\" aria-readonly=\\"false\\" autocomplete=\\"off\\"></div>
                  <div class=\\"v-input__append-inner\\">
                    <div class=\\"v-input__icon v-input__icon--append\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-menu-down theme--light\\"></i></div>
                  </div><input type=\\"hidden\\">
                </div>
                <div class=\\"v-menu\\">
                  <!---->
                </div>
              </div>
              <div class=\\"v-text-field__details\\">
                <div class=\\"v-messages theme--light\\">
                  <transition-group-stub tag=\\"div\\" name=\\"message-transition\\" class=\\"v-messages__wrapper\\"></transition-group-stub>
                </div>
              </div>
            </div>
          </div>
        </div> <button type=\\"button\\" class=\\"ma-8 v-btn v-btn--outlined theme--light v-size--x-large pink--text text--darken-1\\"><span class=\\"v-btn__content\\">
      APLICAR
    </span></button>
      </div>
      <div class=\\"v-navigation-drawer__border\\"></div>
    </aside>
  </div>
</main>"
`;
